src/Microsoft.CodeQuality.Analyzers/Core/Microsoft.CodeQuality.Analyzers.Core.csproj

```diff
@@ -46,7 +46,7 @@
     <EmbeddedResource Update="Documentation\XmlDocumentationCommentsAnalyzersResources.resx">
       <Generator>ResXFileCodeGenerator</Generator>
       <LastGenOutput>XmlDocumentationCommentsAnalyzersResources.Designer.cs</LastGenOutput>
-      <CustomToolNamespace>Documentation\XmlDocumentationComments.Analyzers</CustomToolNamespace>
```

Comment:
Actually, I also need to regenerate the designer.cs file now that the custom tool namespace changed. I'll do it with a separate commit.